pom.xml

```diff
@@ -98,6 +98,12 @@
             <artifactId>b3p-gt2-suf2</artifactId>
             <version>5.0.0-RC2</version>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-wfs-ng</artifactId>
+            <version>19.2</version>
```

Comment:
hier geen versie opgeven (en type is overbodig), de geotools versie wordt beheerd met een property die in de `datastorelinker-bom` wordt beheerd. Ik zal deze daar toevoegen en een nieuwe release maken; deze branch moet dan op de master worden gerebased